### PR TITLE
Changes up nrhm

### PR DESCRIPTION
**Pull request by lwyszomi**
@snopoke

### CODE REVIEW

File: custom/up_nrhm/filters.py [hunk 1]
```diff
@@ -105,7 +105,7 @@ def get_hierarchy(self):
 
 class SampleFormatFilter(BaseSingleOptionFilter):
     slug = 'sf'
-    label = 'Report type'
+    label = ugettext_noop('Report type')
```
Comment (snopoke) [inline custom/up_nrhm/filters.py]: These should all be using `ugettext_lazy`. See http://commcare-hq.readthedocs.org/en/latest/translations.html for details.

**Comment by snopoke** [comment]
You don't need to update the translation files. We have a separate process for that.

Appart from comments this is GTG

**Comment by lwyszomi** [comment]
@snopoke Thanks, updated

File: custom/up_nrhm/reports/asha_facilitators_report.py [hunk 2]
```diff
@@ -1,14 +1,13 @@
 from corehq.apps.reports.datatables import DataTablesHeader, DataTablesColumn
 from corehq.apps.reports.generic import GenericTabularReport
 from corehq.apps.reports.standard import CustomProjectReport, DatespanMixin
-from corehq.apps.reports.filters.dates import DatespanFilter
 from corehq.apps.reports.util import format_datatables_data
-from custom.up_nrhm.filters import DrillDownOptionFilter
 from custom.up_nrhm.sql_data import ASHAFacilitatorsData
+from django.utils.translation import ugettext_noop as _
```
Comment (snopoke) [inline custom/up_nrhm/reports/asha_facilitators_report.py]: This is still `ugettext_noop` (and also others below)

**Comment by lwyszomi** [comment]
@snopoke updated

**Pull request merged**